crates/transaction/manifests/src/manifests_security_shield/transaction_manifest_unsecurified_entity_owner_badge_into_bucket_putting.rs

```diff
@@ -0,0 +1,60 @@
+use profile_supporting_types::AnyUnsecurifiedEntity;
+
+use crate::prelude::*;
+
+impl TransactionManifestUnsecurifiedEntityOwnerBadgeIntoBucketPutting
+    for TransactionManifest
+{
+}
+
+pub trait TransactionManifestUnsecurifiedEntityOwnerBadgeIntoBucketPutting {
+    /// Produces and puts the owner badge of an Unsecurified Entity by calling
+    /// "securify" and put said owner badge in a Bucket and returns the bucket.
+    fn put_owner_badge_in_bucket(
+        builder: ScryptoTransactionManifestBuilder,
+        unsecurified_entity: &AnyUnsecurifiedEntity,
+    ) -> (ScryptoTransactionManifestBuilder, ScryptoManifestBucket) {
+        let is_account = unsecurified_entity.entity.is_account_entity();
+        let owner_badge_bucket_name = "owner_badge_bucket";
+        let owner_badge = if is_account {
+            SCRYPTO_ACCOUNT_OWNER_BADGE
+        } else {
```

Comment:
MINOR: Minor robustness smell here using a boolean - I'd suggest a `enum WalletEntityType { Account, Identity }` instead - so that if this ever changed in future, it'd be a compiler error here.